docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/global-default-private-registry.md

```diff
@@ -2,36 +2,56 @@
 
-This section is about configuring the global default private registry, and focuses on how to configure the registry from the Rancher UI after Rancher is installed.
+:::
 
-For instructions on setting up a private registry with command line options during the installation of Rancher, refer to the [air-gapped installation guide](../../../pages-for-subheaders/air-gapped-helm-cli-install.md).
+A private registry is a private, consistent, and centralized source of truth for the container images in your clusters. You can use a private container image registry to share custom base images within your organization.
 
-If your private registry requires credentials, it cannot be used as the default registry. There is no global way to set up a private registry with authorization for every Rancher-provisioned cluster. Therefore, if you want a Rancher-provisioned cluster to pull images from a private registry with credentials, you will have to [pass in the registry credentials through the advanced cluster options](#setting-a-private-registry-with-credentials-when-deploying-a-cluster) every time you create a new cluster.
+There are two main ways to set up a private registry in Rancher:
 
-## Setting a Private Registry with No Credentials as the Default Registry
+* Set up a global default registry through the **Settings** tab in the global view.
+* Set up a private registry in the advanced options under cluster-level settings. 
+
+The global default registry is intended to be used in air-gapped setups, for registries that don't require credentials. The cluster-level private registry is intended to be used in setups where the private registry requires credentials.
+
+## Set a Private Registry with No Credentials as the Default Registry
 
 1. Log into Rancher and configure the default administrator password.
-1. Click **☰ > Global Settings**.
-1. Go to the setting called `system-default-registry` and choose **⋮ > Edit Setting**.
-1. Change the value to your registry (e.g. `registry.yourdomain.com:port`). Do not prefix the registry with `http://` or `https://`.
+1. Select **☰ > Global Settings**.
+1. Go to `system-default-registry` and choose **⋮ > Edit Setting**.
+1. Enter your registry's hostname and port (e.g. `registry.yourdomain.com:port`). Do not prefix the text with `http://` or `https://`.
+
+**Result:** Rancher pulls system images from your private registry.
+
+### Namespaced Private Registry with RKE2 Downstream Clusters
 
-**Result:** Rancher will use your private registry to pull system images.
+Most private registries should work, by default, with RKE2 downstream clusters.
 
-## Setting a Private Registry with Credentials when Deploying a Cluster
+However, you'll need to do some additional steps if you're trying to set a namespaced private registry whose URL is formated like this: `website/subdomain:portnumber`.
```

Comment:
Line 34 was added to address the concern that urls with port numbers won't automatically work with downstream RKE2 clusters:
>  The issue that we have is that if someone uses a registry url like website/subdomain:5555, it will work for a downstream RKE server, but it will not work for RKE2 by default.